content/en/security/logs.md

```diff
@@ -62,6 +44,17 @@ Additionally, certain features are not available at the moment to customers who
 
 If you have any questions about how the Log Management Service satisfies the applicable requirements under HIPAA, please contact your account manager.
 
+**Note:**
+
+HIPAA-enabled custommer used to require the use of specific endpoints to submit logs in order to enforce some specific encryptions. This is not needed anymore as that those encryptions are now enabled on all log submission endpoints.
```

Comment:
```suggestion
Before version {edit suggestion to add version}, HIPAA-enabled customers were required to use specific endpoints to submit logs. These endpoints enforced HIPPA required encryptions.  For version {edit suggestion to add version} +, these HIPAA required encryptions are enabled on all log submission endpoints.

Reply:
It is not a version change. More a date.

The reason we did not simply remove those endpoints from the page is because we don't want users to think those endpoints do not exist anymore and that their setup is incorrect and they need to change something.

Maybe we can find a better phrasing for this but the idea is to kind of get rid of those legacy endpoints.